components/raftstore-v2/src/operation/command/write/ingest.rs

```diff
@@ -73,6 +73,15 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         if stale_ssts.is_empty() {
             return;
         }
+        let sst_index = self.sst_state().sst_applied_index();
```

Comment:
I don't think this will work well.  It will likely unnecessarily delay the SST cleanup. 
self.sst_state() is Peer/Region level state. Imagining we have two import SSTs calls on the same region.
The first import is with apply index1 and the second is with apply index2.
Then after the second is called, the sst_state is updated with apply index2.  Let's assume apply index1 is flushed and apply index2 is not flushed.
Now when cleanup sst is called,  because  here "log_recovery.as_ref()[off] >= sst_index" will always return false (because sst_index is apply index2, which is not flushed) even for the first import's sstmeta.

Reply:
yes, good suggestion. I wiil record the relationship between the sst meta with the entry index.